lib/standard/stream.nit

```diff
@@ -392,6 +392,29 @@ abstract class BufferedReader
 		return c.ascii
 	end
 
+	# Peeks up to `n` bytes in the buffer
+	#
+	# The operation does not consume the buffer
```

Comment:
What about some nitunits to explain things with examples? Except that StringReader is not a BufferedReader but having a `peek` on it could have been nice :(